addons/godot-next/inspector_plugins/delegation_inspector_plugin.gd

```diff
@@ -0,0 +1,32 @@
+tool
+extends EditorInspectorPlugin
+
+var obj_stack: Array
+
+#warning-ignore:unused_argument
+func can_handle(p_object: Object) -> bool:
+	return true
+
+func parse_begin(p_object: Object) -> void:
+	obj_stack.push_front(p_object)
+	if p_object.has_method("_parse_begin"):
+		p_object._parse_begin(self)
+
+func parse_category(p_object: Object, p_category: String) -> void:
+	if p_object.has_method("_parse_category"):
+		p_object._parse_category(self, p_category)
+
+func parse_property(p_object: Object, p_type: int, p_path: String, p_hint: int, p_hint_text: String, p_usage: int) -> bool:
+	if p_object.has_method("_parse_property"):
+		var pinfo = PropertyInfo.new(p_path, p_type, p_hint, p_hint_text, p_usage)
+		if p_object._parse_property(self, pinfo):
+			return true
```

Comment:
If the parse_property method itself returns boolean, then we should return the result of `p_object._parse_property` if they have the method, rather than automatically returning true.

Reply:
Okay ;)